src/index.html

```diff
@@ -4,11 +4,93 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
+
+    <header class="header">
+      <a class="header_img" href="#">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+
+            <a
+            class="is-active nav__link"
+            href="#apple"
+            >
+            Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            href="#samsung"
+            >
+            Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            href="#smartphones"
+            >
+            Smartphones
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              class="nav__link"
              href="#samsung"
            >
              Samsung
            </a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#smartphones"
            >
              Smartphones
            </a>
          </li>
```

Please fix indents everywhere and add empty line between multiple sibling blocks of html